content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -303,6 +303,9 @@ to get a mapping of UIDs to names for both FlowSchemas and
 PriorityLevelConfigurations.
 
 ## Observability
+
+### Metrics
```

Comment:
need we revise the metrics section to reflect the recent metrics enhancements?

Reply:
Yes.  https://github.com/kubernetes/website/pull/22655